### ISSUE TITLE
Measure avaible parts not working

### ISSUE THREAD

**Issue by mrandt** [jkuusama/LitePlacer-ver2#30]
Function to "Measure available parts on tape" does not work for me (tested SVN Release 18). When I click it from right-click context menu of tape position nothing happens.

I have set the part image using new "photograph part" feature and it works fine for "calibrate via part template (image)".

I noticed that there is also an option "Rescal All Tapes for Available Parts" in menu "Tape". I think there is a typo, it should probably be "Rescan all Tapes". And clicking this function does not do anything for me either.

**Comment by thethereza** [comment]
yeah, issue is that it's a bit tricky setting the detection threshold so it detects parts and not missing parts.  planning on doing some tests and seeing if i can get the system to auto set the threshold by comparing it to an empty location on a tape.

did you try the photograph all components option?
